docs/docusaurus/docs/components/examples_under_test.py

```diff
@@ -483,6 +483,16 @@
         # data_context_dir="",
         backend_dependencies=[],
     ),
+    # Create a custom Action for a Cloud Checkpoint
+    IntegrationTestFixture(
+        # To test, run:
+        # pytest --docs-tests -k "cloud_docs_example_create_a_custom_action" tests/integration/test_script_runner.py
+        name="cloud_docs_example_create_a_checkpoint",
```

Comment:
This test name looks like copy/pasta. It's the same as the name for the test above it. I assume test names should be unique and match the name in the commented out command for running the test

```suggestion
        name="cloud_docs_example_create_a_custom_action",
```